aiohue/v2/controllers/base.py

```diff
@@ -3,6 +3,8 @@
 import asyncio
 from asyncio.coroutines import iscoroutinefunction
 from collections.abc import Callable, Iterator
+
+# pylint: disable=no-name-in-module
```

Comment:
why is this ?

Reply:
Don't know. Probably some lint warning that came up with the original changes still included. I removed it and still passes.